### PR TITLE
Add tooltip timeout.

### PR DESCRIPTION
**Pull request by DinacQuerystorm**
Added timeouts for tooltips.

### CODE REVIEW

File: Controls/ObjectExplorer/ObjectExplorerUC.xaml [hunk 1]
```diff
@@ -100,7 +100,7 @@
                 </TreeView.ItemContainerStyle>
                 <TreeView.ItemTemplate>
                     <HierarchicalDataTemplate ItemsSource="{Binding Nodes, Converter={StaticResource sortConverter}}">
-                        <StackPanel Orientation="Horizontal" ToolTipService.ShowDuration="60000" Margin="2" AllowDrop="True" Drop="StackPanel_Drop" DragOver="StackPanel_DragOver">
+                        <StackPanel Orientation="Horizontal" ToolTipService.InitialShowDelay="500" ToolTipService.ShowDuration="60000" Margin="2" AllowDrop="True" Drop="StackPanel_Drop" DragOver="StackPanel_DragOver">
```
Comment (anakic) [inline Controls/ObjectExplorer/ObjectExplorerUC.xaml]: I think we should remove ToolTipService properties from here, these should be set by the style in the containing application.

File: Controls/PropertiesEditor/PropertyEditorUC.xaml [hunk 2]
```diff
@@ -45,7 +45,7 @@
...
                             <StackPanel Orientation="Horizontal">
                                 <TextBlock Text="{Binding Name, Converter={locconv:StringCapitalizationSeparatorConverter}}" Grid.Column="0" VerticalAlignment="Center" x:Name="lblPropertyName"></TextBlock>
-                                <Image Height="10" Cursor="Help" DockPanel.Dock="Right" Margin="5" RenderOptions.BitmapScalingMode="HighQuality" Source="/Thingie.WPF;component/Resources/help.png" Visibility="{Binding Description, Converter={locconv:MapConverter '_null:Collapsed;_default:Visible'}}" ToolTip="{Binding Description}" ToolTipService.ShowDuration="60000"/>
+                                <Image Height="10" Cursor="Help" DockPanel.Dock="Right" Margin="5" RenderOptions.BitmapScalingMode="HighQuality" Source="/Thingie.WPF;component/Resources/help.png" Visibility="{Binding Description, Converter={locconv:MapConverter '_null:Collapsed;_default:Visible'}}" ToolTip="{Binding Description}" ToolTipService.InitialShowDelay="500" ToolTipService.ShowDuration="60000"/>
```
Comment (anakic) [inline Controls/PropertiesEditor/PropertyEditorUC.xaml]: Ditto

**Review by anakic — COMMENTED** [review]
Remove the tooltip settings from here and see if the tooltip settings of the containing control/app are properly propagated.

File: Controls/ObjectExplorer/ObjectExplorerUC.xaml (hunk 1 above)
Comment (DinacQuerystorm) [reply Controls/ObjectExplorer/ObjectExplorerUC.xaml]: I'm not sure if we can do that for the [ToolTipService](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.tooltipservice?view=netcore-3.1).

> You set the properties of the ToolTipService class by attaching them directly to the element that exposes the tooltip.

I guess I could try and do it by applying the style to the parent element and see what happens?

Comment (anakic) [reply Controls/ObjectExplorer/ObjectExplorerUC.xaml]: Yes please, give it a try. If they're not inherited, see if you can set them in style in the containing application.

**Review by anakic — APPROVED** [review]

**Pull request merged**